src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -147,7 +147,7 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, $value = nul
      *
      * @return ChoiceListView The choice list view
      */
-    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null)
+    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null, $labelAttr = null)
```

Comment:
Yes they all use the same interface.

The BC break is only mentioned as 'Yes' in the description for now, is there anything more I need to do about it ?